src/main/scala/com/softwaremill/realworld/profiles/ProfilesRepository.scala

```diff
@@ -0,0 +1,16 @@
+package com.softwaremill.realworld.profiles
+
+import zio.ZLayer
+
+class ProfilesRepository {
+
+  // TODO user proper db or create in-memory thread-safe store
+  val profiles: Map[Int, StoredProfile] = Map(10 -> StoredProfile(10, "Dummy", "Dummy author", ""))
```

Comment:
nit: the same as above, consider using data that is mentioned in the API spec

Reply:
Discussed offline. We can close it now and go back to it when DB gets introduced.